ts/components/Pinpad.tsx

```diff
@@ -29,14 +39,15 @@ const Pinpad: React.SFC<Props> = props => {
       codeLength={PIN_LENGTH}
       compareWithCode={compareWithCode || ""}
       cellBorderWidth={2}
-      inactiveColor={color(variables.brandLightGray)
+      inactiveColor={color(inactiveColor || variables.brandLightGray)
```

Comment:
I would make `activeColor` and `inactiveColor` required so you don't have to resort to referring to the `variables`, let the parent component decide